hack/api-reference/garden-config.json

```diff
@@ -1,48 +0,0 @@
-{
```

Comment:
Nit: can we remove also the api docs related to the old API?

```
rm hack/api-reference/garden.md
```

Reply:
nice catch, done